### ISSUE TITLE
GraphViz doesn't build pangocairo on Linux

### ISSUE THREAD

**Issue by Keno** [JuliaPackaging/Yggdrasil#3905]
It's there on OS X, but not on Linux, looking through the build, the issue is the following:
```
# pkg-config --cflags "pangocairo >= 1.14.9"
Package xproto was not found in the pkg-config search path.
Perhaps you should add the directory containing `xproto.pc'
to the PKG_CONFIG_PATH environment variable
```

**Comment by Keno** [comment]
Ok, so this is because the xorg headers are marked as a build dependency in https://github.com/JuliaPackaging/Yggdrasil/blob/master/P/Pango/build_tarballs.jl#L44, but we don't propagate build dependencies into dependent packages. I'll fix this by adding the build dependency to GraphViz as well, but we should ideally have a generic way to automatically add any build dependencies from our upstreams (or maybe decide that if the headers are required it's not really a `BuildDependency`).

**Comment by Keno** [comment]
Fixed in #3903.

**Comment by giordano** [comment]
> but we should ideally have a generic way to automatically add any build dependencies from our upstreams (or maybe decide that if the headers are required it's not really a BuildDependency).

Added to https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/639#issuecomment-974914252
